tardis/tardis_portal/static/js/facility_view.js

```diff
@@ -1,265 +1,313 @@
-  // Capitalises the first letter (adapted from http://codepen.io/WinterJoey/pen/sfFaK)
-  app.filter('capitalise', function() {
-      return function(input, all) {
-        return (!!input) ? input.replace(/([^\W_]+[^\s-]*) */g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();}) : '';
-      }
-  });
+// Capitalises the first letter (adapted from http://codepen.io/WinterJoey/pen/sfFaK)
+app.filter('capitalise', function () {
+    return function (input, all) {
+        return (!!input) ? input.replace(/([^\W_]+[^\s-]*) */g, function (txt) {
+            return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+        }) : '';
+    }
+});
 
-  // Filter to produce nice file size formatting (adapted from https://gist.github.com/yrezgui/5653591)
-  app.filter('filesize', function () {
+// Filter to produce nice file size formatting (adapted from https://gist.github.com/yrezgui/5653591)
+app.filter('filesize', function () {
     var units = [
-      'bytes',
-      'KB',
-      'MB',
-      'GB',
-      'TB',
-      'PB'
+        'bytes',
+        'KB',
+        'MB',
+        'GB',
+        'TB',
+        'PB'
     ];
 
-    return function( bytes, precision ) {
-      if ( isNaN( parseFloat( bytes )) || ! isFinite( bytes ) ) {
-        return '?';
-      }
+    return function (bytes, precision) {
+        if (isNaN(parseFloat(bytes)) || !isFinite(bytes)) {
+            return '?';
+        }
 
-      var unit = 0;
+        var unit = 0;
 
-      while ( bytes >= 1024 ) {
-        bytes /= 1024;
-        unit ++;
-      }
+        while (bytes >= 1024) {
+            bytes /= 1024;
+            unit++;
+        }
 
-      return bytes.toFixed( + precision ) + ' ' + units[ unit ];
+        return bytes.toFixed(+precision) + ' ' + units[unit];
     };
-  });
+});
+
 
+app.controller('FacilityCtrl', function ($scope, $resource, $interval, $log) {
 
-  app.controller('FacilityCtrl', function($scope, $http, $interval, $log, $filter) {
+    var countRes = $resource('/facility/fetch_data/:facility_id/count/');
+    var datasetDetailRes = $resource('/facility/fetch_datafiles/:dataset_id/',
+        {}, {
+            'get': {method: 'GET', isArray: true}
+        });
+    var facilityListRes = $resource('/facility/fetch_facilities_list/', {}, {
+        'get': {method: 'GET', isArray: true}
+    });
+    var facilityDataRes = $resource('/facility/fetch_data/:facility_id/:start_index/:end_index/', {
+        start_index: 0,
+        end_index: 50
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Identifier 'end_index' is not in camel case.](https://www.codacy.com/app/mytardis/mytardis/file/1429591694/issues/source?bid=2125231&fileBranchId=2988853#l50)